aldryn_redirects/forms.py

```diff
@@ -58,9 +59,56 @@ def import_from_dataset(dataset):
     return new
 
 
+def validate_row(row, site_domain_id_mapping):
```

Comment:
ideally the cleaning happens on import, this way we don't have to read and process the file twice.
Let's do this at a later stage though.

Reply:
This is SUPER fast the way it is anyway. And I did that way in order to isolate `clean()` and `do_import()`. Conceptually, raising errors on `do_import()` is like raising errors on `model.save()` and should be avoided, IMO.

Reply:
Agreed, I meant more like on import we save what we can and then once done show a page with all the failed entries.

Reply:
ahh gotcha.  =]

Well, this is up to you. Change is simple. Do you want me to work on that?

Reply:
The harder part will be adapt the tests, but it's not that hard anyway

Reply:
nah, let's land it as is for now :)

Reply:
ok!